legacy-sdk/whirlpool/tests/sdk/whirlpools/utils/fetcher-util.test.ts

```diff
@@ -53,18 +65,18 @@ describe("fetcher util tests", () => {
 
   async function initializePositionBundleWithPositions(
     whirlpool: PublicKey,
-    bundleIndexes: number[],
+    bundleIndexes: number[]
```

Comment:
sorry to leave a similar comment again, but worth reverting these formatting changes?